frontend/src/features/mapSideBar/tabs/ltsa/LtsaChargeSubForm.tsx

```diff
@@ -17,35 +18,59 @@ export const LtsaChargeSubForm: React.FunctionComponent<ILtsaChargeSubFormProps>
 }) => {
   const { values } = useFormikContext<LtsaOrders>();
   const charges = getIn(values, withNameSpace(nameSpace, 'chargesOnTitle')) ?? [];
+  const [isExpanded, setIsExpanded] = React.useState<boolean>(true);
+
   return (
     <React.Fragment key={`charge-sub-row-${nameSpace}`}>
-      <StyledSectionHeader>Charges, Liens and Interests</StyledSectionHeader>
-      {charges.length === 0 && 'this title has no charges'}
-      <FieldArray
-        name={withNameSpace(nameSpace, 'chargesOnTitle')}
-        render={({ push, remove, name }) => (
-          <React.Fragment key={`charge-sub-row-${nameSpace}`}>
-            {charges.map((charge: ChargeOnTitle, index: number) => {
-              const innerNameSpace = withNameSpace(nameSpace, `chargesOnTitle.${index}.charge`);
-              return (
-                <React.Fragment key={`charge-sub-row-${innerNameSpace}`}>
-                  <SectionField label="Nature">
-                    <Input field={`${withNameSpace(innerNameSpace, 'transactionType')}`} />
-                  </SectionField>
-                  <SectionField label="Registration #">
-                    <Input field={`${withNameSpace(innerNameSpace, 'chargeNumber')}`} />
-                  </SectionField>
-                  <SectionField label="Registered date">
-                    <Input field={`${withNameSpace(innerNameSpace, 'applicationReceivedDate')}`} />
-                  </SectionField>
-                  <LtsaChargeOwnerSubForm nameSpace={innerNameSpace} />
-                  {index < charges.length - 1 && <hr></hr>}
-                </React.Fragment>
-              );
-            })}
-          </React.Fragment>
+      <StyledSectionHeader>
+        Charges, Liens and Interests
+        {!isExpanded && (
```

Comment:
@Maheshbabu28 this is fine for now, but if you have time this sprint (after your other stories are done) please refactor this into a component. We will likely need collapseable form sections in the future, and a reusable component will be valuable (and reduce code duplication).

Reply:
When you write the component, also add some unit tests for that component if you have time. The tests should check that the state of the arrow icon matches the visibility of the component within the collapse.

Reply:
Sure Devin